gammapy/spectrum/core.py

```diff
@@ -473,12 +473,24 @@ def from_hdulist(cls, hdulist, hdu1='SPECTRUM', hdu2='EBOUNDS'):
         ebounds = fits_table_to_table(hdulist[2])
         emin = ebounds['E_MIN'].quantity
         emax = ebounds['E_MAX'].quantity
+
+        # Check if column are present in the header
+        quality=None
+        areascal=None
+        backscal=None
+        if 'QUALITY' in counts_table:
```

Comment:
This should be 

```
if 'QUALITY' in counts_table.colnames:
```

This is also the reasons why the tests fail ...
@registerrier I tried to push the changes to your branch but apparently I don't have access. Either you grant acess to me or you correct this yourself, up to you :smile: